### ISSUE TITLE
<br> element causes screen readers to pause and repeat the HTML element name

### ISSUE THREAD

**Issue by jonjahr** [adobe/balance-text#147]
### Expected Behaviour
Screen readers should read balanced text normally

### Actual Behaviour
Screen reader reads "Heading level three no matter where we grow.  Heading level three our values remain the same."  Possibly due to the presence of the `<br>` element.
https://user-images.githubusercontent.com/6802815/159798649-9c6714cb-4b79-4323-9c11-ed0593322a3f.mp4

### Reproduce Scenario (including but not limited to)
Our QA vendor reproduced it (and provided the video attached above).  I don't know what screen reader software they used.  

#### Steps to Reproduce
Sorry, I don't have exact steps.  I know can't reproduce this in MacOS with VoiceOver Utility in Safari, Chrome, or Firefox.

#### Platform and Version
balance-text 3.2.1

#### Sample Code that illustrates the problem
```
<h3>No matter where we grow,<br data-owner="balance-text">our values remain the same</h3>
```

#### Logs taken while reproducing problem
None

**Comment by redmunds** [comment]
@jonjahr Thanks for logging bug, and especially the PR to fix it! If you can, please get the name/version of the screen read software from QA vendor and add it in notes for the sake of posterity.

**Comment by redmunds** [comment]
I notice that you logged this bug against v3.2.1, but the latest version is v3.3.0. Is this an IE bug?

FYI, I plan to apply this fix to latest branch and call it v3.3.1, so IE users will not get fix.

**Comment by jonjahr** [comment]
No, not an IE bug. Not sure why I logged it to 3.2.1, my mistake. I’ll reach out to QA vendor. Thanks!
________________________________
